lib/Traits/TraitsClasses/ReferenceTraits.ts

```diff
@@ -0,0 +1,11 @@
+import ModelTraits from "../ModelTraits";
+import primitiveTrait from "../Decorators/primitiveTrait";
+
+export default class ReferenceTraits extends ModelTraits {
+  @primitiveTrait({
+    type: "boolean",
+    name: "Is catalog item open in workbench",
+    description: "Whether the item in the workbench open or collapsed."
+  })
+  isOpenInWorkbench?: boolean = true;
+}
```

Comment:
This is needed so we can see loading spinner in the workbench, while a reference is loading